Lib/pyclbr.py

```diff
@@ -179,21 +180,32 @@ def _readmodule(module, path, inpackage=None):
     return _create_tree(fullmodule, path, fname, source, tree, inpackage)
 
 
+class Stack(list):
+    """Smart list, which adds the ending line of a class/function when it 
+    is removed from the stack (when its scope has ended).
+    """
+    def __init__(self):
```

Comment:
list.__init__ optionally populates the empty list returned by list.__new__.  This subclass init with no parameters does nothing and therefore should not be included.